epilepsy12/constants/causes.py

```diff
@@ -34,22 +34,24 @@
 
 EPILEPSY_GENE_DEFECTS = (
     ("UBE", "UBE3A"),    # 722056009
-    ("GLU", "GLUT1"),
+    ("GLU", "GLUT1"),   # 445252005
     ("SLC", "SLC2A1"),  # 782911008
     ("MEC", "MECP2"),   # 702816000
     ("SCN", "SCN1A"),   # 230437002
     ("STX", "STXBP1"),  # 768666006
     ("CDK", "CDKL5"),   # 773230003
     ("KCN", "KCNQ2"),   # 778001003
     ("SCN", "SCN2A"),   # 778002005
-    ("KCN", "KCNT1"),
```

Comment:
TODO: why am I getting these updates? Potentially I branched off `live` when I should have branched off `development`